Looms.PoS/Looms.PoS.Domain/Daos/ServiceDoa.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Looms.PoS.Domain.Daos;
+
+public record ServiceDao
+{
+    public Guid Id { get; init; }
+    public string Name { get; init; } = string.Empty;
+    public string Category { get; init; } = string.Empty;
+    public string Description {get; init;} = string.Empty;
+
+    [Column(TypeName = "decimal(10,2)")]
+    public decimal Price { get; init; }
+    public int DurationMin { get; init; }
+    public Guid BusinessId { get; init; }
+    public Guid TaxId { get; init; }
+    public bool IsDeleted { get; init; } = false;
+    // Do i need to reference the BusinessDao here?
```

Comment:
yes, every relation should be referenced

Reply:
added the relation